helpers/CssHelper.php

```diff
@@ -24,11 +24,12 @@
 
 class CssHelper
 {
-
+    // phpcs:disable PSR2.Methods.MethodDeclaration
     private static function _buildWarning(): string
     {
         return " /* Do not edit */";
     }
+    // phpcs:enable PSR2.Methods.MethodDeclaration
```

Comment:
This is a private method, can we rename the method as `buildWarning()` and fix the call on line 47 instead of disabling the warning? :)